builders/interoperability/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -823,7 +823,7 @@ unsubscribe();
 
 ### Utility Functions {: #sdk-utils }
 
-The XCM SDK provides three utility functions: `isXcmSdkDeposit`, `isXcmSdkWithdraw`, and XCM UTILS `toDecimal`, and `toBigInt`.
+The XCM SDK provides three utility functions: `isXcmSdkDeposit`, `isXcmSdkWithdraw`, and XCM UTILS `toDecimal`, `toBigInt` and `hasDecimalOverflow`.
```

Comment:
```suggestion
There are utility functions in both the XCM SDK and the XCM Utilities packages. The XCM SDK provides the following SDK-related utility functions:

- `isXcmSdkDeposit`
- `isXcmSdkWithdraw`

And the XCM Utilities package provides the following generic utility functions:

- `toDecimal`
- `toBigInt`
- `hasDecimalOverflow`
```

made this change in a separate PR so, moving it to this one (and adding `hasDecimalOverflow`) as this will probably get merged first